assembly/guide-configs/errors.json

```diff
@@ -0,0 +1,152 @@
+{
+  "file_name": "errors.md",
+  "id": "errors",
+  "file_dir": "references",
+  "title": "Errors",
+  "sidebar_label": "Errors",
+  "description": "This guide is an overview of the errors that can appear on the Temporal Web UI.",
+  "toc_max_heading_level": 4,
+  "add_tabs_support": true,
+  "sections": [
```

Comment:
Would be nice to have 1 context section first that talks about where these errors appear - but not a blocker for this.

Reply:
I'll keep this in mind.